### PR TITLE
[16.0][MIG] website_odoo_debranding

### PR DESCRIPTION
**Pull request by yibudak**

### CODE REVIEW

**Comment by luke-stdev001** [comment]
@yibudak ,

Thanks for your work on this! 

It looks like there is some portal-related branding leftover here:
[https://github.com/odoo/odoo/blob/16.0/addons/portal/views/portal_templates.xml#L223-L235](https://github.com/odoo/odoo/blob/16.0/addons/portal/views/portal_templates.xml#L223-L235)

Is this within the scope of this module?

**Comment by yibudak** [comment]
> @yibudak ,
> 
> Thanks for your work on this!
> 
> It looks like there is some portal-related branding leftover here: https://github.com/odoo/odoo/blob/16.0/addons/portal/views/portal_templates.xml#L223-L235
> 
> Is this within the scope of this module?

image
Hello,
This template is related to portal module and it's debranded with server-brand/portal_odoo_debranding.
https://github.com/OCA/server-brand/blob/14.0/portal_odoo_debranding/views/portal_templates.xml

File: website_odoo_debranding/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+By default the option is enabled, so no additional configuration is needed.
+You can always control the visibility in `Website > Customize > Brand Promotion > Remove Odoo Promotional Link`
```
Comment (tarteo) [inline website_odoo_debranding/readme/USAGE.rst:2]: I can't find this option anymore. Do you know where the optional views are in 16.0?

**Review by tarteo — APPROVED** [review]
Tested it on Runboat, it works.

**Review by petrus-v — APPROVED** [review]

**Pull request merged**